pages/common/join.md

```diff
@@ -18,3 +18,7 @@
 - Produce a line for each unpairable line for file1:
 
 `join -a {{1}} {{file1}} {{file2}}`
+
+- Join two files, the '-' file from standard input:
```

Comment:
Perhaps this better
```suggestion
- Join a file with data from stdin:
```

Reply:
because the '-' file can be file1 or file2, so I think know '-' from stdin is important.

Reply:
may be changed  "Join '-' and file, the '-' from stdin"

Reply:
I think we could simplify this to just be
```suggestion
- Join a file from stdin:
```